universal-application-tool-0.0.1/app/views/components/FieldWithLabel.java

```diff
@@ -241,6 +263,17 @@ public ContainerTag getContainer() {
       ContainerTag textAreaTag = textarea().withType("text").withText(this.fieldValue);
       fieldTag = textAreaTag;
     } else if (this.fieldType.equals("number")) {
+      // Setting inputmode to decimal gives iOS users a more accessible keyboard
+      fieldTag.attr("inputmode", "decimal");
+
+      // Set min and max values for client-side validation
+      if (this.minValue.isPresent()) {
+        fieldTag.attr("min", minValue.getAsLong());
```

Comment:
May want to add a simple test that these are set in the output in test/views/components/FieldWithLabelTest.java

Reply:
done